src/StatisticalAnalyser.java

```diff
@@ -125,4 +125,16 @@ public double pathWeightMean(ArrayList<ArrayList<Path>> paths) {
         stats[1] = Math.sqrt(acc/(double)nPaths); // desvio padrao
         return stats;
     }
+
+    // Calculate routing distance -> all paths lengths / #paths
```

Comment:
Change the return statement so that comment is useless: create a variable to the number of paths. Then get ride of the comment line :)